src/pages/ManhwaFreak/main.ts

```diff
@@ -0,0 +1,100 @@
+import { pageInterface } from '../pageInterface';
+
+export const ManhwaFreak: pageInterface = {
+  name: 'ManhwaFreak',
+  domain: ['https://manhwafreak.com'],
+  database: 'ManhwaFreak',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    if ($('div.chnav.ctop.nomirror').length > 0) {
+      return true;
+    }
+    return false;
+  },
+  isOverviewPage(url) {
+    if ($('div.bigbanner').length > 0) {
+      return true;
+    }
+    return false;
+  },
+  getImage() {
+    return $('section.series > div.info > img').attr('src');
```

Comment:
Too specific, every change to the website would break this. `.series .info img` is probably enough